src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -819,13 +769,12 @@ public void SetRigidity(int rigidity)
             }
 
             rigidity = intRigidity > rigidity ? intRigidity - stepsLeft : intRigidity + stepsLeft;
-            cost = stepsLeft * Constants.MEMBRANE_RIGIDITY_COST_PER_STEP;
         }
 
         var newRigidity = rigidity / Constants.MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO;
-        var prevRigidity = Rigidity;
+        var prevRigidity = CurrentSpecies.MembraneRigidity;
 
-        var action = new MicrobeEditorAction(this, cost, DoRigidityChangeAction, UndoRigidityChangeAction,
+        var action = new MicrobeEditorAction(this, 0, DoRigidityChangeAction, UndoRigidityChangeAction,
```

Comment:
If the costs are dynamically calculated, why keep them at all in MicrobeEditorAction?